changelog.d/383.change.rst

```diff
@@ -0,0 +1,2 @@
+``attr.validators.in_()`` now raises a ``ValueError`` with a useful message even if the options are a string and the value is not a string.
+Previously, this raised a ``TypeError`` (matching e.g. ``1 in "abc"``).
```

Comment:
I cannot quite parse the “matching” part – any chance to rephrase that? I genuinely don’t know what that means and it almost looks like a typo to me. :)

Reply:
How about "(just like `1 in "abc"`)" ?

Reply:
ah yeah maybe drop the `just` and we’re golden